### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by Juanlu001**
Several changes intended to fix some `setup.py` minor problems. These changes have already been tested in the testing PyPI server:

https://test.pypi.org/project/satellogic-quaternions/

Still missing a proper long description though.

### CODE REVIEW

**Comment by Juanlu001** [comment]
Warehouse (new PyPI) still needs some more work to support Markdown:

https://github.com/pypa/warehouse/issues/869#issuecomment-340928703

So I'm leaving it as it stands. This is ready to review & merge.

**Comment by Juanlu001** [comment]
The instructions to do a release are here: https://github.com/satellogic/quaternions/wiki/HOWTO-Release

File: setup.py [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
-    version='0.1.3',
+    name='satellogic_quaternions',
+    version=version["__version__"],
     author='Matias Graña',
     author_email='matias@satellogic.com',
-    long_description='This is a library for dealing with quaternions in python in a unified way.',
+    description='This is a library for dealing with quaternions in Python in a unified way.',
+    long_description=open('README.md').read(),
     packages=find_packages(exclude=["tests"]),
     license="?",
```
Comment (matiasg) [inline setup.py]: good moment to set this as GPL v3

Comment (Juanlu001) [reply setup.py]: Oops, this needs a rebase.

Comment (Juanlu001) [reply setup.py]: Updated.

Comment (matiasg) [reply setup.py]: I just saw this was changed in another PR, :+1:

**Pull request merged**